### DISCUSSION TITLE
auto config question

### DISCUSSION THREAD

**Discussion by bearded-papa** [RandomNinjaAtk/arr-scripts#242] [Q&A] [answered]
I have set the Lidarr, Radarr & Sonarr scripts to auto-config my setup but when I delete a show or movie in plex it doesn't mark the show as unwanted in sonarr, radarr or lidarr.
When I change the option in the three arr's the script resets it the next time it runs.
I have looked through the scripts to see if I can alter this but it is a bit beyond my knowledge and comfort zone to tinker.

How can i resolve this?

**Comment by RandomNinjaAtk** [comment]
Pretty sure deleting a show/movie in Plex and having Plex marking it as unwanted is not a feature that exists. 

The scripts do not change the wanted/unwanted status of things. So I’m not sure what would be your issue.

You can disable auto config after the first run and see if that helps.

**Reply by bearded-papa** [reply]
![Screenshot_20240420_124237_Opera.png](https://github.com/RandomNinjaAtk/arr-scripts/assets/143163316/4f676e96-aa7d-46d8-8919-7d12a72f44ca)

Unmonitored deleted episodes get unselected when the scrips run on radarr and sonarr

**Reply by RandomNinjaAtk** [reply]
That is a sonarr feature, so a problem with sonarr itself.

The only script that messes with wanted/unwanted is the daily episode trimmer of your using that.

**Comment by bearded-papa** [comment]
Now I have removed your scripts the selection isn't being changed so my assumption would be that something within the scripts is changing it

Sent from Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Reply by RandomNinjaAtk** [reply]
Disable the daily trimmer script and see if that fixes it for you. If it does, then it’s just a misunderstanding and misconfiguration on your part…

Also disable autoconfig, because that changes the Arr apps settings

**Reply by RandomNinjaAtk** [reply, marked as answer]
Your issue is that auto config is adjusting the settings, disable it in the conf file.
